docs/api/app.md

```diff
@@ -890,6 +890,15 @@ technologies, such as screen readers, has been detected. See
 https://www.chromium.org/developers/design-documents/accessibility for more
 details.
 
+### `app.setAccessibilitySupportEnabled(enabled)` _macOS_ _Windows_
+
+* `enabled` Boolean - Enable or disable accessibility tree rendering
```

Comment:
Is accessibility support disabled by default? If so, these docs should probably mention that.

For folks who don't know what the "accessibility tree rendering" is, a link would be helpful. Is this the right URL? https://developers.google.com/web/fundamentals/accessibility/semantics-builtin/the-accessibility-tree

Reply:
Yes, it's disabled by default both in Chrome and Electron. The link is super helpful, adding it.